nodes/FormX/actions/document/asyncExtract.operation.ts

```diff
@@ -23,19 +23,22 @@ const displayOptions: IDisplayOptions = {
 export const description = updateDisplayOptions(displayOptions, properties);
 
 export async function execute(this: IExecuteFunctions, i: number): Promise<INodeExecutionData[]> {
-	const imageUrl = this.getNodeParameter('imageUrl', i, undefined, {
-		extractValue: true,
-	});
+	const binaryDataField = this.getNodeParameter('binaryDataField', i);
+
+	let dataBuffer: Buffer;
```

Comment:
Consider adding a func that get binary data field in 1 line as I see this code repeated several times.